src/parser/jobs/rdm/modules/Gauge.js

```diff
@@ -44,30 +42,124 @@ export const SEVERITY_LOST_MANA = {
 	80: SEVERITY.MAJOR,
 }
 
-export const SEVERITY_WASTED_FINISHER = {
-	1: SEVERITY.MINOR,
-	2: SEVERITY.MEDIUM,
-	3: SEVERITY.MAJOR,
-}
-
-const MANA_DIFFERENCE_THRESHOLD = 30
+export const MANA_DIFFERENCE_THRESHOLD = 30
 const MANA_LOST_DIVISOR = 2
-const MANA_CAP = 100
+export const MANA_CAP = 100
 const ENHANCED_SCATTER_GAIN = 8
 const ENHANCED_SCATTER_44_GAIN = 10
-const MISSING_HARDCAST_MANA_VALUE = 11
-const MANAIFCATION_MULTIPLIER = 2
-const MANA_DONT_CAST_THRESHOLD = 96
-const FINISHER_GAIN = 21
-const MELEE_COMBO_COST = 80
+//const MISSING_HARDCAST_MANA_VALUE = 11  //removed with refactor to GaugeAction, change to damage type events prevents need for guessing at missing casts
+const MANAFICATION_MULTIPLIER = 2
+
+class GaugeAction {
```

Comment:
I don't think it would improve readability or reusability to split it out, since the class was created mostly to avoid having an anonymous object for the mana object being added to the fabricated event and to group the mana calculation functions for readability.